### DISCUSSION TITLE
commant to install docker buildx in wolfi os

### DISCUSSION THREAD

**Discussion by Tanmoy037** [wolfi-dev/os#33502] [Q&A]
I am using this docker file - 

FROM gocd/gocd-agent-wolfi:v24.3.0

USER root

# Install Maven
RUN apk add --no-cache maven

# Install OpenJDK 8 and Java 21
RUN apk add --no-cache openjdk-8 openjdk-21

# Set JAVA_HOME to the default Java version (Java 21 in this case)
ENV JAVA_HOME="/usr/lib/jvm/java-21-openjdk"
ENV PATH="$JAVA_HOME/bin:$PATH"

# Install Terraform
RUN apk add --no-cache terraform

#Install Docker
RUN apk add --no-cache docker openrc

RUN apk add docker-buildx-plugin

# Install Python and Pip
RUN apk add --no-cache python3 py3-pip

# Install AWS CLI
RUN apk add aws-cli-v2

# Add AWS credentials (ensure the credentials file exists)
# ADD ./aws.credentials /root/.aws/credentials

# Switch back to GoCD user

---------------------------------------------------------------------
error -

 > [6/8] RUN apk add docker-buildx-plugin:                                                                                                                             
0.135 fetch https://packages.wolfi.dev/os/aarch64/APKINDEX.tar.gz                                                                                                      
2.300 ERROR: unable to select packages:                                                                                                                                
2.300   docker-buildx-plugin (no such package):
2.300     required by: world[docker-buildx-plugin]
------
Dockerfile:21
--------------------
  19 |     RUN apk add --no-cache docker openrc
  20 |     
  21 | >>> RUN apk add docker-buildx-plugin
  22 |     
  23 |     # Install Python and Pip
  
  Which command should solve my problem?

**Comment by Tanmoy037** [comment]
solved that issue !
| RUN apk add --no-cache docker 
getting ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

Any possible solution?

**Comment by xnox** [comment]
Hi!

Please contact gocd publisher for support.

Wolfi & Chainguard publish supported containers on DockerHub only under https://hub.docker.com/u/chainguard 

Wolfi is a container only OS, meaning there is no init system and no daemons start by default. When using docker CLI inside containers, one typically bind-mounts sockets from the host. You should be able to find guides like this online.

Thanks!
